examples/ui_elements/text/escaped.xml.njk

```diff
@@ -13,4 +13,4 @@ tags: text
     <![CDATA[Escaped with <![CDATA[]>:
     <hello>
       <world>]]></text>
-{% endblock %}
+    {% endblock %}
```

Comment:
This seems to be a result of the CDATA block. I wasn't able to get it to stop indenting it with the current content.